transcribed/page1160.txt

```diff
@@ -0,0 +1,23 @@
+001             ;*SYSTEM* JOBS                                                  PAGE 116
+002
+003     117 016 SCORE:   SKIPL SUPCOR
+004     017 002          PUSHJ P,UPL3
```

Comment:
It's difficult to see, but this should be UFLS.  We can verify this against the symbol table which says "UFLS 017 002".

Reply:
Agreed.